IapExample/App.js

```diff
@@ -118,8 +118,6 @@ class Page extends Component {
       await RNIap.initConnection();
       if (Platform.OS === 'android') {
         await RNIap.flushFailedPurchasesCachedAsPendingAndroid();
-      } else {
-        await RNIap.clearTransactionIOS();
```

Comment:
If I remember correctly, this was to help people test consumable purchases. Without this, they should implement checking receipts and then clear transactions. 

Do you still think this should be removed in the example project?

Reply:
I do think it should be removed. People who use this code as base to implement consumable purchases, will find their purchases being cleared for them on relaunch without their knowledge, and won't understand why they don't receive updates for their purchases.
At the very least, a comment should be added that this call will consume all pending transactions, and that they won't get updates on consecutive launches. But I still think removing it is the better option.

Reply:
Thanks for the reply! Could you kindly make some comments without just removing all?
I think people might have questions about how to repurchase consumable products. We used to have many questions on that made us have this code in the example.